plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/ElasticsearchTableHandle.java

```diff
@@ -57,13 +62,15 @@ public ElasticsearchTableHandle(
             @JsonProperty("schema") String schema,
             @JsonProperty("index") String index,
             @JsonProperty("constraint") TupleDomain<ColumnHandle> constraint,
+            @JsonProperty("regexes") Map<String, String> regexes,
             @JsonProperty("query") Optional<String> query,
             @JsonProperty("limit") OptionalLong limit)
     {
         this.type = requireNonNull(type, "type is null");
         this.schema = requireNonNull(schema, "schema is null");
         this.index = requireNonNull(index, "index is null");
         this.constraint = requireNonNull(constraint, "constraint is null");
+        this.regexes = requireNonNull(regexes, "regexes is null");
```

Comment:
do a defensive copy to ensure table handle class remains immutable:
```
ImmutableMap.copyOf(requireNonNull(regexes, "regexes is null"));
```

Reply:
Done